docs/bot_tutorial/message_embeds.md

```diff
@@ -0,0 +1,62 @@
+# Message Embeds
+
+A [Message Embed](https://b1naryth1ef.github.io/disco/api/disco_types_message.html#messageembed) represents a Discord Embed object. An Embed object is used to display certain data, in a more layed out and pretty way than regular Discord messages.
+
+An example of a message embed: 
+
+![alt text](https://i.stack.imgur.com/HRWHk.png "A discord embed")
+
+An embed can contain the following buildstones:
+* Author, including link and avatar
+* Title
+* Description
+* Field(s)
+* Thumbnail image
+* Image
+* Footer text

```

Comment:
In embeds, `footer` is one attribute as `author` is, which you placed all on one line. You should probably do the same here;

> Footer, including text and icon